content/12.privacy.md

```diff
@@ -1,6 +1,7 @@
 ## Tip 10: Don't share models trained on sensitive data {#privacy}
 
 Practitioners may encounter datasets that cannot be shared, such as ones for which there would be significant ethical or legal issues associated with release [@doi:10.1371/journal.pcbi.1005399].
+Examples of such data include classified, confidential, ajd trade secret biological data as well as medical records and personally identifiable information.
```

Comment:
```suggestion
Examples of such data include classified, confidential, trade secret biological data as well as medical records, certain genomic assays, and personally identifiable information [@doi:10.1038/s41576-020-0257-5].
```
typo + add reference for certain types of data in the field that are of special consideration